src/index.js

```diff
@@ -0,0 +1,945 @@
+    eng: {
+      shiftUp: 'm',
+      shiftDown: 'M',
+    },
+    ru: {
+      shiftUp: 'ь',
+      shiftDown: 'Ь',
+    },
+  },
+  {
+    class: ['Comma'],
+    eng: {
+      shiftUp: ',',
+      shiftDown: '<',
+    },
+    ru: {
+      shiftUp: 'б',
+      shiftDown: 'Б',
+    },
+  },
+  {
+    class: ['Period'],
+    eng: {
+      shiftUp: '.',
+      shiftDown: '>',
+    },
+    ru: {
+      shiftUp: 'ю',
+      shiftDown: 'Ю',
+    },
+  },
+  {
+    class: ['Slash'],
+    eng: {
+      shiftUp: '/',
+      shiftDown: '?',
+    },
+    ru: {
+      shiftUp: '.',
+      shiftDown: ',',
+    },
+  },
+  {
+    class: ['ArrowUp', 'special'],
+    eng: {
+      shiftUp: '▲',
+      shiftDown: '▲',
+    },
+    ru: {
+      shiftUp: '▲',
+      shiftDown: '▲',
+    },
+  },
+  {
+    class: ['ShiftRight', 'special'],
+    eng: {
+      shiftUp: 'Shift',
+      shiftDown: 'Shift',
+    },
+    ru: {
+      shiftUp: 'Shift',
+      shiftDown: 'Shift',
+    },
+  },
+];
+
+const keyRow5 = [
+  {
+    class: ['ControlLeft', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+  {
+    class: ['MetaLeft', 'special'],
+    eng: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+    ru: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+  },
+  {
+    class: ['AltLeft', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['Space', 'special'],
+    eng: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+    ru: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+  },
+  {
+    class: ['AltRight', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['ArrowLeft', 'special'],
+    eng: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+    ru: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+  },
+  {
+    class: ['ArrowDown', 'special'],
+    eng: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+    ru: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+  },
+  {
+    class: ['ArrowRight', 'special'],
+    eng: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+    ru: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+  },
+  {
+    class: ['ControlRight', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+];
+
+/* global document, localStorage */
+
+const rows = [keyRow1, keyRow2, keyRow3, keyRow4, keyRow5];
+
+const myBody = document.querySelector('body');
+
+// create wrapper for keyboard
+myBody.innerHTML =
+  '<section class="wrapper"><textarea id="text" placeholder="Write smth!" rows="15"></textarea><section class="keyboard"></section></section>';
```

Comment:
one line code is evil. in case if you are going to insert directly HTML code, please make formatting as well

Reply:
The same problem. It add spaces to the begining of text area:
![image](https://user-images.githubusercontent.com/52204035/69100138-907c4e00-0a6d-11ea-8107-4bd18ac2dd34.png)